src/wechaty/user/contact.py

```diff
@@ -59,6 +59,8 @@
     from .message import Message
     # pytype: disable=pyi-error
     from .url_link import UrlLink
+    from .mini_program import MiniProgram
```

Comment:
I notice that if coverage decrease, `x` will be show in commit. Maybe i should add a pr to fix it?
![image](https://user-images.githubusercontent.com/4636271/163317103-b3c3510d-8f5a-4ab3-bf27-a8da93a99e81.png)